gerrymandering/gerrymandering.ipynb

```diff
@@ -46,22 +48,15 @@
    "outputs": [],
    "source": [
     "import holoviews as hv\n",
-    "from holoviews import opts\n",
+    "import hvplot.dask # noqa\n",
+    "import hvplot.pandas # noqa\n",
```

Comment:
Is it necessary to import `hvplot.pandas`? I think since the data is stored as a dask dataframe and you've already imported the `hvplot.dask`, you don't need it.
```suggestion
```